src/adapter/src/coord/appends.rs

```diff
@@ -52,6 +49,12 @@ pub(crate) struct DeferredPlan {
     pub plan: Plan,
 }
 
+#[derive(Clone)]
+pub(crate) enum BuiltinTableUpdateSource {
+    DDL,
+    Background,
+}
```

Comment:
Doccomments on here to communicate the meaning.

Reply:
Added